Edit project system docs (#215)
microsoft/VSExtensibility: docs/new-extensibility-model/extension-guides/project/project.md

    .QueryAsync(cancellationToken);
```

> Here we are getting all recursively nested solution folders. Note that `VisualPath` the path as it shows up in Solution Explorer.

Reviewer: ```suggestion
Here we are getting all recursively nested solution folders. `VisualPath` is the path as it shows up in Solution Explorer.
```
